mlflow/sagemaker/cli.py

```diff
@@ -340,9 +340,10 @@ def push_model_to_sagemaker(
 @click.option("--build/--no-build", default=True, help="Build the container if set.")
 @click.option("--push/--no-push", default=True, help="Push the container to AWS ECR if set.")
 @click.option("--container", "-c", default=IMAGE, help="image name")
+@cli_args.SKIP_JAVA
```

Comment:
Could we reuse INSTALL_JAVA instead of creating a new option here? :)

Reply:
Yeah, the one thing is I think the default should be different. When you have a model artifact you're building an image for, and you thus have a model flavor, you can detect automatically whether or not you need java. So, in that case it is safe to set a default of False for `--install-java`. However, I don't think that is the right default for sagemaker, where the image is intended to serve arbitrary model flavors.

Put another way, using the same flag and the same default would mean that this change is a breaking change for sagemaker build-and-push-container.

So, we could:
* Have the same option, with the same defaults (breaking change)
* Same option, different default (not a breaking change)

Reply:
Sorry missed this. Could we make default value of install_java to be None, and modify correspondingly :)